source/_integrations/alexa.smart_home.markdown

```diff
@@ -221,33 +237,771 @@ alexa:
 
 - There are different endpoint URLs, depending on the region of your skill. Please check the available endpoints at <https://developer.amazon.com/docs/smarthome/send-events-to-the-alexa-event-gateway.html#endpoints>
 - The `client_id` and `client_secret` are not the ones used by the skill that have been set up using "Login with Amazon" (in the [Alexa Developer Console][amazon-dev-console]: Build > Account Linking), but rather from the "Alexa Skill Messaging" (in the Alexa Developer Console: Build > Permissions > Alexa Skill Messaging). To get them, you need to enable the "Send Alexa Events" permission.
 - If the "Send Alexa Events" permission was not enabled previously, you need to unlink and relink the skill using the Alexa App, or else Home Assistant will show the following error: "Token invalid and no refresh token available. Also, you need to restart your Home Assistant after each disabling/enabling the skill in Alexa."
 
-### Alexa web-based app
+#### Configure Filter
+
+By default, no entity will be excluded. To limit which entities are being exposed to Alexa, you can use the `filter` parameter. Keep in mind only [supported components](#supported-integrations) can be added.
+
+{% raw %}
+```yaml
+# Example filter to include specified domains and exclude specified entities
+alexa:
+    smart_home:
+      filter:
+        include_domains:
+          - alarm_control_panel
+          - light
+        exclude_entities:
+          - light.kitchen_light
+```
+{% endraw %}
+
+Filters are applied as follows:
+
+1. No includes or excludes - pass all entities
+2. Includes, no excludes - only include specified entities
+3. Excludes, no includes - only exclude specified entities
+4. Both includes and excludes:
+   * Include domain specified
+      - if domain is included, and entity not excluded, pass
+      - if domain is not included, and entity not included, fail
+   * Exclude domain specified
+      - if domain is excluded, and entity not included, fail
+      - if domain is not excluded, and entity not excluded, pass
+      - if both include and exclude domains specified, the exclude domains are ignored
+   * Neither include or exclude domain specified
+      - if entity is included, pass (as #2 above)
+      - if entity include and exclude, the entity exclude is ignored
+      
+See the [troubleshooting](#troubleshooting) if for issues setting up the integration.
+
+
+#### Alexa Display Categories
+
+Configure a display category to override the display category and iconography each entity is shown in the Alexa app. This makes it easier to find and monitor devices.
+
+```yaml
+light.kitchen_light:
+  display_categories: LIGHT,SWITCH
+```
+
+<div class='note info'>
+Devices such as cameras, doorbells, garage doors, and alarm control panels require specific display categories to provide all available features from Amazon Alexa. Overriding the display category provided by Home Assistant will limit features provided by Amazon Alexa. 
+</div> 
```

Comment:
@frenck Regarding HTML usage; is this still acceptable? Is there a more preferred way? Or just remove these type of notes?

Reply:
This is currently acceptable and documented in our developer documentation.

Reply:
@frenck Thanks, must have overlooked it since it was under the `Create a new page` section.